src/localization/authStrings.js

```diff
@@ -13,6 +13,8 @@ export const authStrings = new LocalizedStrings({
     repeat_password: 'Please re-enter password',
     user_name: 'User Name',
     email: 'E-mail Address',
+    warning_sync_edit:
+      'WARNING:\nThis will effect syncing for this store.\nMake sure you know what you doing!',
```

Comment:
Left alignment of `<Text/>` makes this warning look a bit funny, but fine for rough pass